src/client/game.cpp

```diff
@@ -1349,6 +1349,8 @@ bool Game::createClient(const GameStartData &start_data)
 	/* Skybox
 	 */
 	sky = new Sky(-1, m_rendering_engine, texture_src, shader_src);
+	if (g_settings->getBool("enable_dynamic_shadows"))
+		sky->setSkyBodyOrbitTilt(rangelim(g_settings->getFloat("shadow_sky_body_orbit_tilt"), 0.0f, 60.0f));
```

Comment:
This code gets called once when the client is created, Sky instance is the cache, and I've changed Game::updateShadows() to use the value from Sky instead of g_settings.